### DISCUSSION TITLE
how to change font size?

### DISCUSSION THREAD

**Discussion by networkoctopus** [isontheline/pro.webssh.net#1188] [Q&A] [answered]
I tried to search for this - I don't see how to change the terminal font size.  Despite there being a help article, it does not show you how.  On macOS I can zoom in and out, but not iOS.

Please can you show me where I can do this?

**Comment by isontheline** [comment, marked as answer]
Hello @dwyeraidan 👋 

You can pinch in / out the terminal to change font size

You can also change it from inside your connection settings.

**Comment by networkoctopus** [comment]
Cheers man. Too easy.  I thought I’d seen it in the ui at some point.. but
couldn’t find it again! didn’t look in the connection settings. Mark as
solved please. 🙏🏻
